manifests/i/IDRIX/VeraCrypt/1.26.7/IDRIX.VeraCrypt.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: IDRIX.VeraCrypt
+PackageVersion: 1.26.7
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  Silent: /qn ACCEPTLICENSE=YES
+  SilentWithProgress: /qb ACCEPTLICENSE=YES
+ElevationRequirement: ElevatesSelf

```

```suggestion
ElevationRequirement: elevatesSelf
```
I think that while the property name has to be PascalCase, the value has to be camelCase.